bears/general/SpaceConsistencyBear.py

```diff
@@ -29,6 +30,8 @@ def run(self,
                                           of tabs.
         :param allow_trailing_whitespace: Whether to allow trailing whitespace
                                           or not.
+        :param allow_trailing_blanklines: Whether to allow trailing blanklines
+                                          at EOF or not.
```

Comment:
Btw this is not the correct docstring indentation ;P

Reply:
Well for now, it is 😉